pkg/cephmgr/collectors/health.go

```diff
@@ -35,7 +35,7 @@ import (
 var (
 	recoveryIORateRegex    = regexp.MustCompile(`(\d+) (\w{2})/s`)
 	recoveryIOKeysRegex    = regexp.MustCompile(`(\d+) keys/s`)
-	recoveryIOObjectsRegex = regexp.MustCompile(`(\d+) objects/s`)
+	recoveryIOObjectsRegex = regexp.MustCompile(`(\d+) services/s`)
```

Comment:
this file should not have been touched at all and this change will actually break the metrics collector.  This looks like a victim of global search and replace.  Please revert all these stray changes from objects to services.